src/js/types/trezor/protobuf.js

```diff
@@ -690,8 +702,8 @@ export type CardanoSignTx = {
     network_id: number,
     certificates: CardanoTxCertificateType[],
     withdrawals: CardanoTxWithdrawalType[],
-    metadata?: string,
     validity_interval_start?: number,
+    auxiliary_data?: CardanoTxAuxiliaryDataType,
```

Comment:
As above (why move this to the end? it's not a new addition, it just replaces metadata).

Reply:
Moved - https://github.com/vacuumlabs/connect/pull/12/commits/cda61468472cd09c479f875e8b2d7b80923ad4e9.